addons/account/models/account_move.py

```diff
@@ -4703,277 +4703,317 @@ def _mail_track(self, tracked_fields, initial):
     # RECONCILIATION
     # -------------------------------------------------------------------------
 
-    def _prepare_reconciliation_partials(self):
-        ''' Prepare the partials on the current journal items to perform the reconciliation.
-        /!\ The order of records in self is important because the journal items will be reconciled using this order.
-
-        :return: a tuple of 1) list of vals for partial reconcilation creation, 2) the list of vals for the exchange difference entries to be created
-        '''
-        def get_odoo_rate(line):
-            return recon_currency._get_conversion_rate(company_currency, recon_currency, line.company_id, line.date)
+    @api.model
+    def _prepare_reconciliation_partial(self, debit_vals, credit_vals):
```

Comment:
You should change the name.
Having 2 method names with only one `s` difference in the same model is confusing

Reply:
What do you suggest then?